src/stringfilter.cpp

```diff
@@ -117,6 +117,19 @@ void StringFilter::AddLine(const char *str)
 	}
 }
 
+/**
+ * Pass another text line from the current item to the filter.
+ *
+ * You can call this multiple times for a single item, if the filter shall apply to multiple things.
+ * Before processing the next item you have to call ResetState().
+ *
+ * @param str Another line from the item.
+ */
+void StringFilter::AddLine(const std::string &str)
```

Comment:
Since you added this, maybe remove the `.c_str()` on the 5 cases that could now use this function?

Reply:
Yes, although it would make more sense to me as a separate PR?

Reply:
Now #10743. 🙂